content/content-strategy/manage-content-requests/index.yml

```diff
@@ -0,0 +1,17 @@
+pagetitle: Manage content requests
+description: Know who's involved and what needs to be done to manage business-ass-usual content requests.
+theme: blue
+header:
+  - /_shared/globalheader.md
+  - /_shared/header.md
+main:
+  - intro.md #attribution
+  - content-lifecycle-image.md #diagram
```

Comment:
Missing a reference to a table of contents. Needs a new `toc.md` and for it to be listed here before any actual content.

Reply:
@shad0wMe0w not sure I follow why a main topic needs a toc.md...?